spec/features/movies_feature_spec.rb

```diff
@@ -25,420 +25,417 @@
-        youtube_id = '73829hsuhf'
-        sign_in_user(admin_user)
-        visit(movie_path(movie))
-        fill_in 'trailer', with: youtube_id
-        click_button('add-trailer-btn')
-        url = URI.parse(current_url)
-        expect("#{url}").to include('trailer-section') #redirects to anchor tag
-        expect(movie.reload.trailer).to eq(youtube_id) #updates the trailer
-      end
-
-      scenario 'non-admin should not see trailer button' do
-        sign_in_user(user)
-        visit(movie_path(movie))
-        expect(page).not_to have_selector('#add-trailer-btn')
-      end
-
-      scenario "update movie button retrieves latest info from API" do
-        sign_in_user(admin_user)
-        fargo
-        visit(movie_path(fargo))
-        expect(fargo.runtime).to eq(90)
-        VCR.use_cassette('update_movie') do
-          click_link("update_movie_link_movie_show")
+      describe "movie show page functionality" do
+        scenario "users can visit the movie show page, which has a slugged url" do
+          sign_in_user(user)
+          visit(movie_path(movie))
+          url = URI.parse(current_url)
+          expect("#{url}").to include("#{movie.slug}")
         end
-        expect(page).to have_content("1h 38m")
-      end
 
-      context "the movie on the show page is on one of the user's lists" do
-        before(:each) do
-          page.driver.browser.manage.window.resize_to(1280,800)
+        scenario "movie show page has genres that are links that filter movies" do
           sign_in_user(user)
           listing
+          visit(movie_path(movie))
+          click_link "Crime"
+          expect(page).to have_selector("#modal_link_#{movie.tmdb_id}")
         end
 
-        scenario "users can add tags to a movie from the movie show page", js: true do
+        scenario "clicking director name goes to director results" do
+          sign_in_user(user)
+          movie = create(:movie_in_tmdb)
           visit(movie_path(movie))
-          fill_in "tag_list", with: "dark comedy, spooky"
-          click_button "add_tags_button_movies_partial"
-          expect(page).to have_content("dark-comedy")
-          expect(page).to have_content("spooky")
-        end #user can tag movie
+          click_link "#{movie.director} (director)"
+          expect(page).to have_content(movie.director)
+        end
 
-        scenario "user can remove tags from the movie show page", js: true do
+        scenario "movie show page has the movie's poster image" do
+          sign_in_user(user)
+          listing
           visit(movie_path(movie))
-          fill_in "tag_list", with: "dark comedy"
-          click_button "add_tags_button_movies_partial", match: :first
-          expect(page).to have_content("dark-comedy")
-          click_link "remove_tag_link_movies_partial"
-          expect(page).not_to have_content("dark-comedy")
+          expect(page).to have_css("img[src*='https://image.tmdb.org/t/p/w185#{movie.poster_path}']")
```

Comment:
This is the only other change in this file, and it's just to make it less brittle since I changed the way the factory handles the image.